aksel.nav.no/website/app/_ui/props-seksjon/PropsSeksjon.tsx

```diff
@@ -58,36 +68,92 @@ function PropTable({ component }: { component: PropsSeksjonComponentT }) {
       <dl>
         {propList.map((prop) => (
           <div className={styles.propsSeksjonRow} key={prop.name}>
-            <Box as="dt" paddingBlock="space-8 0" paddingInline="space-8">
-              <Code as="h4" highlighted>{`${prop.name}${
-                prop?.required ? "" : "?"
-              }`}</Code>
-            </Box>
-            <BodyShort as="dd">
-              <Box as="ul" overflowX="auto">
-                <PropsSeksjonCode code={prop.type} title="Type" wrap />
-                <PropsSeksjonCode
-                  /* We assume that if type starts with ", its an union-type */
-                  code={
-                    (prop.type ?? "").startsWith('"')
-                      ? `"${prop.defaultValue}"`
-                      : prop.defaultValue
-                  }
-                  title="Default"
-                  wrap
-                />
-                <PropsSeksjonDescription
-                  description={prop.description}
-                  params={prop.params}
-                />
-                <PropsSeksjonCode code={prop.example} title="Example" />
-              </Box>
-            </BodyShort>
+            <PropEntry prop={prop} />
           </div>
         ))}
       </dl>
     </div>
   );
 }
 
+const PropEntry = ({
+  prop,
+}: {
+  prop: NonNullable<
+    NonNullable<PropsSeksjonComponentT["propref"]>["proplist"]
+  >[0];
+}) => {
+  if (prop.deprecated) {
+    return (
+      <details>
+        <summary>
+          <Box
+            as="dt"
```

Comment:
Details inside `dl`-list is not valid a11y 💥

Reply:
yeah, did away with those now 👍